test/test_ops.py

```diff
@@ -725,13 +725,13 @@ def func(x_, offset_, mask_, weight_, bias_):
             return ops.deform_conv2d(x_, offset_, weight_, bias_, stride=stride,
                                      padding=padding, dilation=dilation, mask=mask_)
 
-        gradcheck(func, (x, offset, mask, weight, bias), nondet_tol=1e-5)
+        gradcheck(func, (x, offset, mask, weight, bias), nondet_tol=1e-5, fast_mode=False)
```

Comment:
fast_mode is False by default, why this change? Also the title says turn on fast gradcheck?

Reply:
WAIT NOT READY YET!

Reply:
Should have marked it as draft :) Now we are good.